examples/oembed.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" />
```

Comment:
Why have you changed `//` to `http://` here and in other places of code? `//` looks like a more general approach that will work in a wider range of environments. For instance, I'm trying to access the `oembed.html` page bu HTTPs in my experiments.

Reply:
To get ability to user open this page locally and test with local instance of alerta.
Without this fix, user opens page with "file:///" protocol and external scripts tries to open with "file:///" protocol too.
If this page is not test page, but it is an example page - I will revert this fixes.

Reply:
Yes, please revert this change.